import-export-cli/cmd/secret/create.go

```diff
@@ -75,12 +78,14 @@ func init() {
 	secretCreateCmd.Flags().StringVarP(&inputPropertiesfile, "from-file", "f", "", "Path to the properties file which contains secrets to be encrypted")
 	secretCreateCmd.Flags().StringVarP(&outputType, "output", "o", "console", "Get the output in yaml (k8) or properties (file) format. By default the output is printed to the console")
 	secretCreateCmd.Flags().StringVarP(&encryptionAlgorithm, "cipher", "c", "RSA/ECB/OAEPWithSHA1AndMGF1Padding", "Encryption algorithm")
+	secretCreateCmd.Flags().StringVarP(&namespace, "namespace", "n", "default", "Namespace for the K8s cluster")
```

Comment:
Shall we make the default value to an empty string,
Someone can change the default namespace to another namespace other than "default".
`k config set-context --current --namespace kube-system`, if we provide empty string, then kubectl resolve it to default namespace of the context. @pubudu538

Reply:
Simply we can remove namespace usage in here and create k8s artifacts without the namespace. When deploying, users can use any namespace or change context as mentioned above.

Reply:
Yes, I think what @pubudu538 mentioned is make sense. We should create the secret yaml file without any namespace then the user can apply it to any namespace as required.